test/act-rules/act-runner.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 const fs = require('fs');
 const http = require('http');
 const handler = require('serve-handler');
-const AxeBuilder = require('@axe-core/webdriverjs');
+const { AxeBuilder } = require('@axe-core/webdriverjs');
```

Comment:
Recent version of `AxeBuilder` required adding `default` to the require or destructuring the object.